### ISSUE TITLE
Give the item durability

### ISSUE THREAD

**Issue by polykatt** [BaKYCoder/mc-backtobed-mod#18]
Maybe some durability as a slight nerf to the item. Maybe enough for 20 uses or something not sure :) or maybe less than 20 and ender pearls as a repair item in anvil. no enchanting though

**Comment by BaKYCoder** [comment]
Thank you for your idea! I’ve actually been considering this feature myself and I will do my best to implement it in a future update.

**Comment by johalun** [comment]
I just came here to suggest this :)  Making durability configurable would be awesome! For our mod pack I'd probably set it to 3 uses.

**Comment by BaKYCoder** [comment]
> I just came here to suggest this :) Making durability configurable would be awesome! For our mod pack I'd probably set it to 3 uses.

Yes, I have a plan to make the mod as configurable as possible so players can customize it for themselves.
